package.json

```diff
@@ -89,13 +89,13 @@
     "@react-native-clipboard/clipboard": "^1.13.2",
     "@react-native-community/netinfo": "^11.3.1",
     "@react-native-cookies/cookies": "^6.2.1",
-    "@react-native-firebase/analytics": "^17.5.0",
-    "@react-native-firebase/app": "^17.5.0",
-    "@react-native-firebase/auth": "^17.5.0",
-    "@react-native-firebase/database": "^17.5.0",
-    "@react-native-firebase/dynamic-links": "^17.5.0",
-    "@react-native-firebase/messaging": "^17.5.0",
-    "@react-native-firebase/remote-config": "^17.5.0",
+    "@react-native-firebase/analytics": "19.1.0",
+    "@react-native-firebase/app": "19.1.0",
+    "@react-native-firebase/auth": "19.1.0",
+    "@react-native-firebase/database": "19.1.0",
+    "@react-native-firebase/dynamic-links": "19.1.0",
+    "@react-native-firebase/messaging": "19.1.0",
+    "@react-native-firebase/remote-config": "19.1.0",
```

Comment:
should we temporarily disable renovate for firebase packages? Guess we can also let it open PRs and we can manually close, assuming it will be blocked by CI

Reply:
doesn't this change effectively disable renovate by specifying that we want the exact versions of these packages?

the CI failed to catch the crash actually (the previous version of this PR which used a higher version of these packages passed) - i think because the sms retrieval functionality which gets the phone number from the sim card on an emulator doesn't work the same as on a real device. i could consistently repro the crashes on my physical device.

but yes i agree, it was my intention to stop renovate from automatically updating these packages without us knowing (it's pretty bad versioning practice that there is a major version upgrade in an underlying android sdk but on the react native project, it is included in a patch version upgrade)